packages/cli/__tests__/zosfiles/__unit__/create/like/__snapshots__/ds.definition.unit.test.ts.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Please rename this folder from "create/like" to "create/ds" to make it consistent with the command name. Sorry for not noticing this earlier.